tests/foreman/api/test_operatingsystem.py

```diff
@@ -101,8 +91,5 @@ def test_create_with_minor(self):
 
         """
         minor = gen_integer(0, 9999999999999999)  # max len: 16 chars
```

Comment:
Pretty strange boundaries:) as code can generate valid and invalid values in the same time. Or I missed something :)